generators/generator-base.js

```diff
@@ -341,7 +341,7 @@ module.exports = class extends PrivateBase {
                     fullPath +
                     chalk.yellow('. Reference to ')}npm dependency (name: ${name}, version:${version})${chalk.yellow(' not added.\n')}`
             );
-            this.debug('Error:', e);
+            this.dnaebug('Error:', e);
```

Comment:
??

Reply:
I checked on the codebase and I don't find any reference for the method `dnaebug`. I missed something?

Reply:
My 4 year-old most probably is really excited with js as much as his dad :rofl:

Reply:
ha ha